### PR TITLE
Gym Tab

### PR DESCRIPTION
**Pull request by zzhikai**
https://trello.com/c/xQ0NoL60/2-create-tabs
Mobile View:
Screenshot 2022-01-26 at 2 19 43 AM
Desktop View:
Screenshot 2022-01-26 at 2 20 59 AM

### CODE REVIEW

**Comment by zzhikai** [comment]
[Create tabs](https://trello.com/c/xQ0NoL60/2-create-tabs)

File: frontend/src/components/GymNavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
```
Comment (gunbux) [inline frontend/src/components/GymNavBar.tsx:1]: can remove React

File: frontend/src/routes/Home/index.tsx [hunk 2]
```diff
@@ -177,6 +178,7 @@ export default function Home() {
             </ActionButton>
           ))}
         </ButtonGroup>
+        <GymNavBar />
```
Comment (gunbux) [inline frontend/src/routes/Home/index.tsx:181]: remove this part since it shouldn't be in the home page?

**Review by gunbux — APPROVED** [review]
minor stuff but otherwise lgtm ^_^

File: frontend/src/components/GymNavBar.tsx (hunk 1 above)
Comment (zzhikai) [reply frontend/src/components/GymNavBar.tsx:1]: > src/components/GymNavBar.tsx
  Line 31:5:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 32:7:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 33:9:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 42:13:  'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 43:15:  'React' must be in scope when using JSX  react/react-in-jsx-scope

It caused some compiler error, any experience with this? Else I will keep it like this first

Comment (gunbux) [reply frontend/src/components/GymNavBar.tsx:1]: O ya i think rhapp is not updated to latest so we still have to keep it in

**Pull request merged**